src/style.css

```diff
@@ -0,0 +1,79 @@
+@import "./blocks/cards.css";
+@import "./blocks/stars.css";
+
+body {
+  margin: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  --blue-color: #00acdc;
+  --text-color: #060b35;
+  --secondary-text-color: #616070;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 24px;
+  margin-left: 3px;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: var(--text-color);
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--secondary-text-color);
+  margin: 0;
+  margin-bottom: 17px;
+}
+
+.card__product-review {
+  display: inline-block;
+  font-size: 10px;
+  color: var(--text-color);
+  margin-left: 18px;
+  margin-top: 2px;
+}
+
+.card__price-word {
+  color: var(--secondary-text-color);
+  font-size: 12px;
+}
+
+.card__price-number {
+  color: var(--text-color);
+  font-size: 16px;
+  font-weight: 700;
+  margin-left: 82px;
+}
+
+.card__button-buy {
+  box-sizing: border-box;
+  display: inline-block;
+  background-color: var(--blue-color);
+  width: 166px;

```

Comment:
do not hardcode margin-left and width for you elements (as well as height)
add paddings for the whole card and make positioning not hardcoded

Reply:
Can you please assist with this one? How can I set positioning for 2 inline elements without margin ? If I could to wrap this 2 elements in div ,then set flex + justify-content: space-between , it could help . But according tasks , I can to create only to div , first one for whole card and second one for stars